modules/@ergonode/channels/src/pages/channels/_channel/_id.vue

```diff
@@ -35,6 +35,11 @@ export default {
         store,
         params,
     }) {
+        await store.dispatch('dictionaries/getInitialDictionaries', {
+            keys: [
+                'channels',
+            ],
+        });
```

Comment:
use `mapActions('dictionaries',...)` as in the previous `_id` file.

Reply:
It has its own purpose to be like that -> different time calling.